src/pages/pricing.tsx

```diff
@@ -26,22 +26,22 @@ const plans: PricingBoxProps[] = [
     {
         title: 'Personal',
         img: <object tabIndex={-1} data={LightBulb}/>,
-        price: isEurope() ? '€8' : '$9',
+        price: <>{isEurope() ? '€8' : '$9'}<span>/month</span></>,
```

Comment:
I suggest adding spaces around the `/`, to make it "breathe" a bit more (and be consistent with `/ month` below).

```suggestion
        price: <>{isEurope() ? '€8' : '$9'}<span> / month</span></>,
```